scripts/lighthouse/home-stats.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
```

Comment:
Good practice: you can add #! /usr/bin/node at the beginning exactly like in Shell scripts to make it an executable script

This way you can do `./home-stats.js` and omit the `node ./home-stats.js`

Reply:
Also you may want to add an `audit` script in `package.json` that simply runs this script.

Haven't tested yet but I am very eager to see the result!

Reply:
You could also add this script to the "Release" doc so we remember to run it regularly